README.md

```diff
@@ -25,198 +25,120 @@ API documentation is available [here](https://input-output-hk.github.io/cardano-
-| cardano-address key public --without-chain-code
-stake_vk16apaenn9ut6s40lcw3l8v68xawlrlq20z2966uzcx8jmv2q9uy7qau558d
-```
-Notice the `stake_vk` prefix to identify a stake verification (public) key.
-
-> :information_source: `1852H/1815H/0H/2/0` is the derivation path that is typically used by Cardano wallet to identify a stake key within HD wallet. If you seek compatibility with Daedalus or Yoroi, use this as well!
 
+> :information_source: The last segment in the path is the key index and can be incremented up to `2^31-1` to derive more keys.
 </details>
 
 <details>
-  <summary>How to generate a payment address from key credential</summary>
+  <summary>How to generate a script verification key (<strong>script.xvk</strong>)</summary>
 
+```console
+$ cardano-address key child 1852H/1815H/0H/3/0 < root.xsk | cardano-address key public --with-chain-code > script.xvk
+script_xvk1mg7xae48d7z4nntd35tey0jmclxaavwmk3kw2lkkt07p3s3x3yy45805manx2kj2neg40kfpy9em36vnkjfm4fw09k66837unrvd70qj75eg0
 ```
-  $ cardano-address recovery-phrase generate --size 15 \
-  | cardano-address key from-recovery-phrase Shelley | tee root.prv
-
-  root_xsk1fqf5s0s7mluhs8q8xduvhy9rgxgj90x8yg0ht92ymhr775k3uap4zxvh4fzcqtg7j9ewvag3lsgswryav3w650gul4mza65nrp6vdx5dt8xelgx87avxcwfqwkajm7masntd3ef3duwj39pnacupl68u4yw4ss9h
-
-  $ cat root.prv \
-  | cardano-address key child 1852H/1815H/0H/0/0 | tee addr.prv
 
-  addr_xsk1krnkg5uyxgdxxpkzrwscr99d0ln9dylg84qspktsvgj42c73uaplrzymsrvef3qsq0mqy8hjxk9m00k9yexym3578lxzszfanmr83cu9jhm9nzmwnfnps36gcz4d328cz49eqtgq4whl2ev402f5ae0ht5kenmd2
-
-  $ cat addr.prv \
-  | cardano-address key public --with-chain-code \
-  | cardano-address address payment --network-tag testnet
-
-  addr_test1vq3a2mkasqfmsc5y5tguuckqaxa3pafrxs9x26d2hqvjehc8dk63q
-```
+> :information_source: The last segment in the path is the key index and can be incremented up to `2^31-1` to derive more keys.
 </details>
 
 <details>
-  <summary>How to generate a delegation address from stake key credentials</summary>
-
-  Follow the steps from 'How to generate a payment address'. Then, simply extend
-  an existing payment address with a stake key!
-
-```
-  $ cat root.prv \
-  | cardano-address key child 1852H/1815H/0H/2/0 \
-  | tee stake.prv
-
-  stake_xsk1pr95hal8lfp4wnq4m9zmvv9emy9hakx0h73qh5zkzeprxek3uapayt7qhelgfgvxx40wqr0g9j5rfwg56ujvlc5thfhrctkqld3gegyc24jf0jp59qkk8xcgqztdp762wmcx5km4w8duh77xjfx6m3yw6qw65q6w
-
-  $ cat addr.prv \
-  | cardano-address key public --with-chain-code \
-  | cardano-address address payment --network-tag testnet \
-  | cardano-address address delegation $(cat stake.prv | cardano-address key public --with-chain-code)
-
-  addr_test1qq3a2mkasqfmsc5y5tguuckqaxa3pafrxs9x26d2hqvjehesytwfe2p0tt06y5ptxwvvm6lfu255dny7yz8e9kv4e83qwwzy29
+  <summary>How to generate a payment address from a payment key (<strong>payment.addr</strong>)</summary>
+  
+```console
+$ cardano-address address payment --network-tag testnet < payment.xvk > payment.addr
```

Comment:
For consistency with previous examples (e.g. this one "How to generate a payment verification key (addr.xvk)"): 
```suggestion
$ cardano-address address payment --network-tag testnet < addr.xvk > payment.addr
```